recipes/chamois/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "CHAMOIS" %}
+{% set name_pypi = "chamois-tool" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.org/packages/source/c/{{ name_pypi }}/{{ name_pypi | replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 4812d2209b3867236892197348ce9c6ddecf7e7444207d4ced10a2068d898b8e
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  run_exports:
+    - {{ pin_subpackage("chamois", max_pin="x.x") }}
+  entry_points:
+    - chamois = chamois.cli:run
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools >=39.2
+    - wheel >=0.30
+  run:
+    - python >=3.7
+    - importlib_resources >=1.0  #[py37, py38, py39]
+    - numpy >=1.16,<3.0
+    - anndata >=0.8,<0.10
+    - gb-io >=0.3.1,<0.4
+    - lz4 >=4.0,<5.0
+    - numpy >=1.0,<3.0
+    - pandas >=1.3,<3.0
+    - platformdirs >=3.0,<5.0
+    - pyhmmer >=0.11.0,<0.12.0
+    - pyrodigal >=3.0,<4.0
+    - rich >=12.4.0
+    - rich-argparse >=1.1,<2.0
+    - scipy >=1.4,<2.0
+    - scikit-learn >=1.0,<2.0
+    - kennard-stone >=2.1,<3.0
+    - rdkit >=2023.3
```

Comment:
add `psutil` as per the failed tests

Reply:
That's a transitive dependency from PyHMMER, it's not used here; guess this issue is `conda` incorrectly getting the latest `psutil` even though it's not supported, this should be fixed in https://github.com/bioconda/bioconda-recipes/blob/master/recipes/pyhmmer/meta.yaml rather than here, I'll open a PR.